TestProjects/Forms/Example001CSharp/Example.iOS/Example.iOS.csproj

```diff
@@ -22,8 +22,20 @@
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
     <MtouchArch>i386</MtouchArch>
-    <MtouchLink>None</MtouchLink>
-    <MtouchDebug>true</MtouchDebug>
+    <MtouchLink>SdkOnly</MtouchLink>
```

Comment:
And same comment: I tihnk it's better to have a working example out of the box.  With that said, if you prefer I disable MTouchLink (SdkOnly) in debug mode, let me know and I'll roll it back.